x-pack/plugins/observability_solution/observability_onboarding/common/telemetry_events.ts

```diff
@@ -7,8 +7,16 @@
 
 import { type EventTypeOpts } from '@elastic/ebt/client';
 
+interface ObservabilityOnboardingIntegrationTelemetryFields {
+  installSource: string;
+  pkgName: string;
+  pkgVersion: string;
+  title: string;
+}
+
 export const OBSERVABILITY_ONBOARDING_TELEMETRY_EVENT: EventTypeOpts<{
```

Comment:
I wonder if we should split this into two separate event types to have the current generic event that just tracks steps along the flow and then another event type that tracks details about a specific flow. `integrations` field only applies to auto-detect, then Firehose flow will have its own set of specific fields, k8s an OTel might have something specific as well. Having everything in one schema will probably get messy over time. WDYT?

Reply:
Agree, just added 88e20111332c3ebfd76c1faca2287970b2545cca to define a custom event type.